src/components/Dashboard.tsx

```diff
@@ -57,6 +58,11 @@ const TemplateComponent: React.FC<{ template: TemplateInList }> = ({ template })
       ? USER_ROLES.APPLICANT
       : USER_ROLES.REVIEWER
 
+  const [loadingFilters, setLoadingFilters] = useState(false)
+  const setLoading = (loadingState:boolean) => {
+    setLoadingFilters(loadingState)
+  }
+
```

Comment:
Do you need this extra function? You should be able to just pass `setLoadingFilters` down to each component.